posh-git.psm1

```diff
@@ -38,7 +38,7 @@ $poshGitPromptScriptBlock = $null
 
 $currentPromptDef = if ($funcInfo = Get-Command prompt -ErrorAction SilentlyContinue) { $funcInfo.Definition }
 
-# HACK: If prompt is missing, create a global one we can overwrite with Set-Item
+# HACK: If prompt is missing, create a global one we can overwrite with Set-Item

```

Comment:
Apparently I [committed](https://github.com/dahlbyk/posh-git/commit/bd70123320f84bac803eed9ca1d54771b186df14) a LF here instead of CRLF:
<img width="377" alt="notepad _2017-01-08_00-32-42" src="https://cloud.githubusercontent.com/assets/133987/21747829/3595a268-d53a-11e6-8d26-b77445bc73f5.png">

VS Code is fixing it. #336 will prevent this type of change in the future.